platform-coop-toolkit/coop-map-directory-index
# feat: add full profile view for organizations (resolves #6)

## Description

Adds proper layout and formatting for full organizational profiles.

TODO:

- [x] Selectively hide various data depending on `Organization.type` ([ref](https://docs.google.com/spreadsheets/d/1CIwTlZMucPz82AVpx-FmLf3AXZlz4oMZp8H387KeSAs/edit))
- [x] Implement "Contact" tab

## Steps to test

1. Check out this branch.
2. Update and build front-end assets: `npm ci && npm run build:production` 
3. Visit some organizational profiles (for type: Cooperative, type: Potential cooperative, type: Shared platform, and type: Supporting organization).

**Expected behavior:** Profiles match designs.

## Additional information

Not applicable.

## Related issues

Blocked by #107.
Resolves #6.
